### PR TITLE
fix(dialog): add wrapper div

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Review by yinonov — COMMENTED** [review]
Are snapshot changes as expected?

**Comment by rachelbt** [comment]
> Are snapshot changes as expected?

the Safari one is obvious, but the otheres that changed it because the last `vwc-dialog` in the ui-test has position relative and now it actually work. + I don't know how it passed before now it has the max-inline-size working.

File: change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "fix(dialog): add wrapper div",
```
Comment (yinonov) [inline change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json:3]: can this be more descriptive? I don't know what to review

Comment (rachelbt) [reply change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json:3]: remember the issue with overflow hidden and elevation filter in safari? So it happens in dialog as well. I added a wrapper thet gets the elevation bg and the drop-shadow filter and the dialog element gets the overflow hidden.

Comment (yinonov) [reply change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json:3]: oh ok, will be good to describe it shortly in the comment

Comment (rachelbt) [inline change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json:3]:
```suggestion
  "comment": "fix(dialog): add wrapper div to dialog to solve elevation shadow issue in Safari",
```

File: change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "fix(dialog): add wrapper div to dialog to solve elevation shadow issue in Safari",
```
Comment (yinonov) [inline change/@vonage-vivid-2bd4cbfa-6367-441c-b142-3387fda80105.json:3]:
```suggestion
  "comment": "add wrapper div to dialog to solve elevation shadow containment issue in Safari",
```

**Review by yinonov — DISMISSED** [review]

**Review by yinonov — APPROVED** [review]

File: libs/components/src/lib/dialog/dialog.template.ts [hunk 3]
```diff
@@ -78,7 +78,6 @@ export const DialogTemplate: (
 	definition: FoundationElementDefinition
 ) => ViewTemplate<Dialog> = () => html<Dialog>`
 	<vwc-elevation dp="12">
-		<!--adding a wrapper div to solve Safari issue with filter and overflow-->
```
Comment (yinonov) [inline libs/components/src/lib/dialog/dialog.template.ts]: I would keep it and even link to the spec. @YonatanKra objection here?

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**